tools/caml_tex.ml

```diff
@@ -614,7 +619,7 @@ let process_file file =
         | Toplevel -> true in
       let global_expected = try Output.expected @@ matched_group 4 !input
         with Not_found -> Output.Ok in
-      start true tex_fmt main [string_of_mode mode];
+      start tex_fmt main [string_of_mode mode];
```

Comment:
Given that `?newline=false` is the default choice, there is a change of behavior here. Is it expected? (I haven't reviewed the code around to know that it matters -- or not.)

Reply:
I found an ever simpler option: since the backends are now less fussy about newline characterd, I emit newlines unconditionally .